packages/pwa/app/commerce-api/index.js

```diff
@@ -117,44 +110,39 @@ class CommerceAPI {
                     get: function(obj, prop) {
                         if (typeof obj[prop] === 'function') {
                             return (...args) => {
+                                const fetchOptions = args[0]
                                 const {locale, currency} = self._config
 
-                                if (args[0].ignoreHooks) {
+                                if (fetchOptions.ignoreHooks) {
                                     return obj[prop](...args)
                                 }
 
-                                return self.willSendRequest(prop, ...args).then((newArgs) => {
-                                    let {sendLocale = true, sendCurrency = false} = apiConfigs[key]
-
-                                    // By default we send the local param and don't send the currency param.
-                                    // You can modify when these are send by using a boolean value send the
-                                    // currency/locale for all calls for a given API, or define an array listing
-                                    // the API's methods you want the currency/locale information to be sent.
-                                    sendLocale = Array.isArray(sendLocale)
-                                        ? sendLocale.includes(prop)
-                                        : !!sendLocale
-                                    sendCurrency = Array.isArray(sendCurrency)
-                                        ? sendCurrency.includes(prop)
-                                        : !!sendCurrency
-
-                                    // Ensure we don't send the pseudo locale.
-                                    if (locale === 'en-XB') {
-                                        sendLocale = false
-                                    }
```

Comment:
Removing this block, because `locale` will never be `en-XB`, unless the page url contains it. In other words, this file was not aware of the pseudo locale.

When creating a new instance of our Commerce API client, we passed in the locale found in the page url.